### PR TITLE
🌐 [i18n-KO] Translated `image_feature_extraction.md` to Korean

### PR DESCRIPTION
**Pull request by mreraser**
# What does this PR do?

Translated the `image_feature_extraction.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

May you please review this PR?
@jun048098, @yijun-lee, @mreraser, @shinhyunji36, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

Hello @stevhliu !  May you please review this PR? Thank you! 

Have a nice day :)

### CODE REVIEW

File: docs/source/ko/tasks/image_feature_extraction.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 이미지 특징 추출[[image-feature-extraction]]
+
+[[open-in-colab]]
+
+이미지 특징 추출은 주어진 이미지에서 의미론적으로 의미 있는 특징을 추출하는 작업입니다. 이에는 이미지 유사성 및 이미지 검색 등 다양한 사용 사례가 있습니다. 게다가 대부분의 컴퓨터 비전 모델은 이미지 특징 추출에 사용할 수 있으며, 여기서 작업 특화 헤드(이미지 분류, 물체 감지 등)를 제거하고 특징을 얻을 수 있습니다. 이러한 특징은 가장자리 감지, 모서리 감지 등 더 높은 수준에서 매우 유용합니다. 또한 모델의 깊이에 따라 실제 세계에 대한 정보(예: 고양이가 어떻게 생겼는지)를 포함할 수도 있습니다. 따라서 이러한 출력은 특정 데이터 세트에 대한 새로운 분류기를 훈련하는 데 사용할 수 있습니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_feature_extraction.md:21]:
```suggestion
이미지 특징 추출은 주어진 이미지에서 의미론적으로 의미 있는 특징을 추출하는 작업입니다. 이는 이미지 유사성 및 이미지 검색 등 다양한 사용 사례가 있습니다.
게다가 대부분의 컴퓨터 비전 모델은 이미지 특징 추출에 사용할 수 있으며, 여기서 작업 특화 헤드(이미지 분류, 물체 감지 등)를 제거하고 특징을 얻을 수 있습니다. 이러한 특징은 가장자리 감지, 모서리 감지 등 고차원 수준에서 매우 유용합니다.
또한 모델의 깊이에 따라 실제 세계에 대한 정보(예: 고양이가 어떻게 생겼는지)를 포함할 수도 있습니다. 따라서 이러한 출력은 특정 데이터 세트에 대한 새로운 분류기를 훈련하는 데 사용할 수 있습니다.
```
개행과 일부 단어를 매끄럽게 바꿔 보았습니다.

File: docs/source/ko/tasks/image_feature_extraction.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+pipe = pipeline(task="image-feature-extraction", model_name="google/vit-base-patch16-384", device=DEVICE, pool=True)
+```
+
+`pipe`를 사용하여 추론하려면 두 이미지를 모두 전달하세요.
+
+```python
+outputs = pipe([image_real, image_gen])
+```
+
+출력에는 두 이미지의 풀링된 임베딩이 포함되어 있습니다.
```
Comment (heuristicwave) [inline docs/source/ko/tasks/image_feature_extraction.md:58]:
```suggestion
출력에는 두 이미지의 통합된(pooled) 임베딩이 포함되어 있습니다.
```
아래 풀링과의 일관성 때문에 제안이 고민되지만, pooled의 의미를 더 잘 전달하기 위해 바꿔보았습니다.

**Review by heuristicwave — COMMENTED** [review]
@mreraser 가장 빠르게 문서를 잘 번역해주셔서 감사합니다!

**Comment by mreraser** [comment]
> @mreraser 가장 빠르게 문서를 잘 번역해주셔서 감사합니다!

소중한 리뷰 감사합니다 😄 @heuristicwave

**Review by mreraser — COMMENTED** [review]
셀프 리뷰 완료

File: docs/source/ko/tasks/image_feature_extraction.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+pipe = pipeline(task="image-feature-extraction", model_name="google/vit-base-patch16-384", device=DEVICE, pool=True)
+```
+
+`pipe`를 사용하여 추론하려면 두 이미지를 모두 전달하세요.
+
+```python
+outputs = pipe([image_real, image_gen])
+```
+
+출력에는 두 이미지의 통합된(pooled) 임베딩이 포함되어 있습니다.
```
Comment (shinhyunji36) [inline docs/source/ko/tasks/image_feature_extraction.md:60]:
```suggestion
출력에는 두 이미지의 풀링된 임베딩이 포함되어 있습니다.
```

Comment (shinhyunji36) [reply docs/source/ko/tasks/image_feature_extraction.md:60]: 원문에 "The output contains pooled embeddings of those two images." 라고 적혀 있고, 이후 번역에서도 pool을 풀링이라고 쓰셨기에 이렇게 쓰는 것은 어떠한지 제안드려 봅니다.

**Review by shinhyunji36 — CHANGES_REQUESTED** [review]

**Comment by shinhyunji36** [comment]
고심해서 번역하신 것이 느껴지네요! 많이 배웠습니다. :) 번역하느라 고생 많으셨습니다 😃 @mreraser

**Review by yijun-lee — APPROVED** [review]
LGTM

File: docs/source/ko/tasks/image_feature_extraction.md (hunk 3 above)
Comment (mreraser) [reply docs/source/ko/tasks/image_feature_extraction.md:60]: 고민해봤는데 원문을 최대한 살리고, 일관적인게 좋은 것 같아서 `풀링된(pooled)`을 하는게 나을 것 같습니다! 리뷰 감사드립니다!

**Review by stevhliu — APPROVED** [review]
LGTM, great job! 🤗

**Pull request merged**